var/spack/repos/builtin/packages/hdfview/package.py

```diff
@@ -15,6 +15,11 @@ class Hdfview(Package):
     homepage = "https://www.hdfgroup.org/downloads/hdfview/"
     url = "https://support.hdfgroup.org/ftp/HDF5/releases/HDF-JAVA/hdfview-3.1.4/src/hdfview-3.1.4.tar.gz"
 
+    version(
+        "3.1.3",
+        sha256="566807495305a32d3da65539a529f268fcfc375e6da32ed07c2b5723b9aa0c66",
+        url="https://support.hdfgroup.org/ftp/HDF5/releases/HDF-JAVA/hdfview-3.1.3/src/hdfview-3.1.3.tar.gz",
+    )
```

Comment:
Can you reorder this (e.g. so this new version appears between `3.1.1` and `3.1.4`? Also, the URL does not look different in format from the package-level URL so I don't think you need to define it here.